src/core/processDirectory.ts

```
@@ -0,0 +1,85 @@
+                })),
+            );
+        }
+
+        console.log('───────────────────────────────────────────────────────');
+        console.log(
+            `Summary: ${summary.renamed} files ${options.doWrite ? 'renamed' : 'would be renamed'}, ` +
+                `${summary.skipped} skipped, ${summary.errors} errors`,
+        );
+
+        return summary;
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)